CPS-XXXXX/README.md

```diff
@@ -0,0 +1,112 @@
+---
+CPS: ?
+Title: ADA token supply exhaustion
```

Comment:
```suggestion
Title: Ada token supply exhaustion
```
I've corrected the PR title according to official capitalisation (`ADA` is the trading symbol, not the token): https://cardano.org/what-is-ada